ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/ant_contrib/tasks/main.yml

```diff
@@ -11,7 +11,7 @@
         path: /opt/apache-ant-1.9.9/lib/ant-contrib.jar
       register: antcontrib
 
-    - name: "ant-contrib.jar installed, skipping download"
+    - name: "Ant-contrib.jar installed, skipping download"
```

Comment:
Suggest prefixing this with something, since `ant-contrib.jar` is the name of a file so should not be adjusted like this.